notesapi/management/commands/create_index.py

```diff
@@ -1,9 +1,34 @@
+from optparse import make_option
+from django.conf import settings
 from django.core.management.base import BaseCommand
-from annotator.annotation import Annotation
+from elasticutils.contrib.django import get_es
+from notesapi.v1.models import NoteMappingType
 
 
 class Command(BaseCommand):
-    help = 'Creates the mapping in the index.'
+    """
+    Indexing and mapping commands.
+    """
+    help = 'Creates index and the mapping.'
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--drop',
+            action='store_true',
+            dest='drop',
+            default=False,
+            help='Recreate index'),
+    )
 
     def handle(self, *args, **options):
-        Annotation.create_all()
+        if options['drop']:
+            # drop existing
+            get_es().indices.delete(index=settings.ES_INDEXES['default'], ignore=404)
+        get_es().indices.create(
+            index=settings.ES_INDEXES['default'],
+            body={
+                'mappings': {
+                    NoteMappingType.get_mapping_type_name(): NoteMappingType.get_mapping()
+                }
+            },
+            ignore=400  # ignore when present
```

Comment:
> maybe we should not ignore 400 when drop=True?

I think it is better not to have ignores at all.
I removed 400 and 404 code ignores from this command.